apps/web/tests/subscribers.spec.ts

```diff
@@ -18,5 +18,5 @@ test('should display subscribers list', async ({ page }) => {
     'Created At',
     'Data',
   ]);
-  await subscribersPage.assertSubscribersTableRowCount(1);
+  await subscribersPage.assertSubscribersTableRowCount(2);
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ How did the nature of this test change?

Reply:
That's a great question. I will delve deeper.